src/main/java/bio/terra/service/status/StatusService.java

```diff
@@ -50,13 +45,6 @@ public StatusService(
   public RepositoryStatusModel getStatus() {
     RepositoryStatusModel statusModel = new RepositoryStatusModel();
 
-    // Used by Unit test: StatusTest
```

Comment:
This was a joy to remove!

We previously needed it because we only tested this code in a test class called `StatusTest`, which was a controller unit test.  By breaking testing apart into `UnauthenticatedApiControllerTest` -- a controller unit test -- and `StatusServiceTest` -- we can test this fault behavior in the former and test the service code in the latter.

Reply:
If you remove this, can/should you also remove `FailLivenessCheck` in clienttests?

Reply:
Great idea.  I wasn't sure if those served a different purpose, but we now cover the needed check in unit tests so it should be removed.

Reply:
Dug a little more and found where the fault was originally added:

https://github.com/DataBiosphere/jade-data-repo/pull/634

It was meant to help test Stairway flight recovery on k8s pod restart.  But later, we upgraded Stairway and removed our custom initialization code, so I feel good about fully removing the fault and all tests:

https://github.com/DataBiosphere/jade-data-repo/pull/1109